ServerManager/classes/GameSave.php

```diff
@@ -178,7 +178,7 @@ public function createZip(GameSession $gameSession): bool
     {
         $urlBase = ServerManager::getInstance()->getAbsoluteUrlBase();
         if (getenv('DOCKER') !== false) {
-            $urlBase = 'http://caddy:80'.ServerManager::getInstance()->getAbsolutePathBase();
+            $urlBase = 'http://localhost:80'.ServerManager::getInstance()->getAbsolutePathBase();
```

Comment:
so before from the "php" container we had to request towards the caddy container using http://caddy, But now we can just use "localhost"